assets/sass/local.sass

```diff
@@ -123,7 +123,7 @@
   &:focus
     outline: 3px solid #ffbf47
 
-#main-content
+.cat-fit-content
```

Comment:
Weird that this is shown on the PR, might need the epic branch updated?

Reply:
i have updated the epic branch, but it is showing as merge. should be gone once i push it to main

Reply:
resolved now